tests/dummy/app/styles/app.css

```diff
@@ -1,3 +1,8 @@
+#ember-testing {
+  height: 100% !important;
+  width: 100% !important;
+  transform: scale(1) !important;
+}
```

Comment:
So I do need to figure out this.  Essentially the css was the following, which was making it particularly difficult with our existing scroll functions in the offset acceptance test.

```
height: 200%
width: 200%
transform: scale(0.5);
```

Let me see if I can bring this back.